package/MDAnalysis/coordinates/ParmEd.py

```diff
@@ -174,9 +174,10 @@ def convert(self, obj):
             ag_or_ts = obj.atoms
         except AttributeError:
             if isinstance(obj, base.Timestep):
-                ag_or_ts = obj.copy()
+                raise ValueError("Writing Timesteps to ParmEd "
+                                 "objects is not supported")
             else:
-                raise_from(TypeError("No Timestep found in obj argument"), None)
+                raise TypeError("No atoms found in obj argument") from None
```

Comment:
raise_from is needed here unfortunately - py2.7 things

Reply:
🐍🐍🐍🐍🐍🐍🐍